app/views/content/life-as-a-teacher/explore-subjects/art-and-design/_article.html.erb

```diff
@@ -0,0 +1,108 @@
+    text: "People may not realise how much potential there is for progression in teaching. They think they go into the world of the classroom and that's it. Lots of teachers have added responsibilities they enjoy doing and find fulfilling.",
+    name: "Chris Farr, teacher training adviser",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+      <p>Teaching art and design opens the door for your pupils to pursue exciting careers in fields as diverse as fashion and textiles, to illustration and video game design.</p>
+      <p>By thinking creatively, your pupils could make significant contributions to these diverse and impactful areas.</p>
+      <p>Your pupils will learn more from you than you might think.  As their teacher, you'll guide them though the skills they need to show their understanding of art and design. You'll teach them valuable life skills such as critical thinking, adaptability and communication.</p>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-art-and-design-programmes-of-study/national-curriculum-in-england-art-and-design-programmes-of-study">national curriculum for art and design</a>.</p>
+    <p>Themes you'll cover when you teach 11 to 14 year olds (key stage 3) include:</p>
+    <ul>
+      <li>observation and recording techniques</li>
+      <li>media exploration</li>
+      <li>proficiency in material handling</li>
+      <li>creativity and idea development</li>
+      <li>contextual understanding</li>
+    </ul>
+  </section>
+
+  <section class="col col-720">
+    <p>In key stage 4 (14 to 16 years olds) you will help pupils demonstrate skills in:</p>
+    <ul>
+      <li>composition and design</li>
+      <li>making appropriate use of colour, line, tone, texture, shape and form, in response to an idea, theme or brief</li>
+      <li>explain and critically review their work throughout</li>
+  </section>
+
+  <section class="col col-720">
+    <p>At A level and AS level (key stage 5), you’ll help pupils develop a deeper knowledge and understanding of:</p>
+    <ul>
+      <li>how ideas, feelings and meanings can be conveyed and interpreted in images and artefacts</li>
+      <li>continuity and change in different genres, styles and traditions</li>
+      <li>recording experiences and observations, in a variety of ways</li>
+      <li>exploring relevant resources; analyse, discuss and evaluate images, objects and artefacts; and make and record independent judgements</li>
+      <li>using knowledge and understanding of the work of others to develop and extend thinking and inform their own work</li>
+      <li>generate and explore potential lines of enquiry using appropriate media and techniques</li>
+  </section>
+
```

Comment:
<p>Teaching pupils at different ages and abilities means you'll always be expanding your knowledge and approach to meet pupils' needs. Once you have qualified teacher status (QTS) you can also teach any other subject, as long as you can show you have enough expertise.</p>
</section>